react/src/index.tsx

```diff
@@ -159,7 +159,11 @@ customElements.define(
 
     return (
       <DefaultProviders {...props}>
-        <Flex direction="column" align="stretch" style={{ minWidth: 200 }}>
+        <Flex
+          direction="column"
+          align="stretch"
+          style={{ minWidth: 200, maxWidth: 310 }}
```

Comment:
You don't need to set the maxWidth. When you set an explicit number in your code, there might be a better way to do the same thing without a hardcoded number. @gee05053  @ironAiken2 

```suggestion
          style={{ minWidth: 200, flex:1 }}
```

Reply:
Oh! My above review was incorrect. The suggestion I provided doesn't work. The explicit width applies only to `backend-ai-react-resource-group-select`, which is used only in `backend-ai-resource-monitor`, so the current code is okay. @gee05053  @ironAiken2  I resolved this conversation.

Reply:
@yomybaby 
Yes😂 I tried yesterday to modify code according your suggestion, and I checked It was not work.
But I agreed your suggestion too about `there might be a better way without a hardcoded number`
I searched other way to solve this issue, and I found solution .
when I removed `wrap` in this [code](https://github.com/lablup/backend.ai-webui/blob/14c9bb563f928fd225c8950c811665f193524984/src/components/backend-ai-resource-monitor.ts#L661), this issue was resolved.
Is this way not good instead of using maxWidth?

Reply:
@yomybaby If you want to check code, please check recent code that I pushed😊